src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import classNames from 'classnames';
+import { FilteredBy } from '../../types/FilteredBy';
+
+type Props = {
+  filterBy: FilteredBy,
+  setFilterBy: (FilteredBy: FilteredBy) => void,
+  isActiveTodos: boolean,
+  clearCompletedTodo: () => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  filterBy,
+  setFilterBy,
+  isActiveTodos,
+  clearCompletedTodo,
+}) => {
+  const handleClick = (value: FilteredBy) => {
+    if (filterBy !== value) {
+      setFilterBy(value);
+    }
+  };
+
+  const handleAllClick = () => handleClick(FilteredBy.ALL);
+  const handleActiveClick = () => handleClick(FilteredBy.ACTIVE);
+  const handleCompletedClick = () => handleClick(FilteredBy.COMPLETED);
```

Comment:
It could be one handler
```suggestion

  const onFilterChange = (filter: FilteredBy) => () => {
    if (filterBy !== value) {
      setFilterBy(value);
    }
  }
```